clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -1988,6 +1989,15 @@ public void testDescribeClientMetricsConfigs() throws Exception {
         }
     }
 
+    @Test
+    public void testCreateDescribeConfigsByDuplicateResources() {
+        ConfigResource resource = new ConfigResource(ConfigResource.Type.BROKER, "1");
+        ConfigResource duplicateResource = new ConfigResource(ConfigResource.Type.BROKER, "1");
+        try (AdminClientUnitTestEnv env = mockClientEnv()) {
+            assertDoesNotThrow(() -> env.adminClient().describeConfigs(asList(resource, duplicateResource)));
```

Comment:
It would be better to test the response size too. Hence, we can reuse the existent test case. for example: 

https://github.com/apache/kafka/blob/16e2b68b73426fc6b77efa9a820afadc9b412627/core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala#L2662

we can add a duplicate `ConfigResource`.

Reply:
Cool, I will do that 👌🏾